Utils/src/GenParticlesProducer.cc

```diff
@@ -35,16 +35,19 @@ class GenParticlesProducer : public edm::global::EDProducer<> {
 private:
     void produce(edm::StreamID, edm::Event&, const edm::EventSetup&) const override;
     const reco::GenParticle* findLast(const reco::GenParticle& particle) const;
-    void saveChain(int depth, int parentId, int parent_idx, const reco::GenParticle& particle, std::unique_ptr<std::vector<math::PtEtaPhiELorentzVector>>& genParticle_vec,
-                   std::unique_ptr<std::vector<int>>& PdgId_vec, std::unique_ptr<std::vector<int>>& Status_vec,
-                   std::unique_ptr<std::vector<int>>& Parent_vec, std::unique_ptr<std::vector<int>>& ParentId_vec,
-                   std::unordered_set<const reco::Candidate *>& stored_particles_ref,
-                   std::vector<const reco::Candidate *>& stored_particles_list, std::vector<const reco::Candidate *>& parents_list) const;
+    void saveChain(int, int, int, const reco::GenParticle&,
```

Comment:
prefer to keep variable names in member function definitions so the interface is more readable/understandable

Reply:
Ugh! You're one of those! ;) Okay, I've added the names back in.